http.json

```diff
@@ -1,1459 +1,1455 @@
-                    "data_type": "string",
-                    "column_order": 1,
-                    "column_name": "File Transferred",
-                    "contains": [
-                        "file path"
-                    ],
-                    "example_values": [
-                        "http://192.168.0.1:8001/web_storage/test_repo/file.tgz"
-                    ]
-                },
-                {
-                    "data_path": "action_result.message",
-                    "data_type": "string",
-                    "column_order": 2,
-                    "column_name": "Result Message",
-                    "example_values": [
-                        "File sent: http://192.168.0.1:8001/web_storage/test_repo/file.tgz"
-                    ]
-                },
-                {
-                    "data_path": "summary.total_objects",
-                    "data_type": "numeric",
-                    "example_values": [
-                        1
-                    ]
-                },
-                {
-                    "data_path": "summary.total_objects_successful",
-                    "data_type": "numeric",
-                    "example_values": [
-                        1
-                    ]
-                }
-            ],
-            "versions": "EQ(*)"
-        }
-    ],
-    "pip_dependencies": {
-        "wheel": [
-            {
-                "module": "beautifulsoup4",
-                "input_file": "wheels/py3/beautifulsoup4-4.9.1-py3-none-any.whl"
-            },
-            {
-                "module": "certifi",
-                "input_file": "wheels/py3/certifi-2022.6.15-py3-none-any.whl"
-            },
-            {
-                "module": "chardet",
-                "input_file": "wheels/shared/chardet-3.0.4-py2.py3-none-any.whl"
-            },
-            {
-                "module": "decorator",
-                "input_file": "wheels/py3/decorator-5.1.1-py3-none-any.whl"
-            },
-            {
-                "module": "idna",
-                "input_file": "wheels/shared/idna-2.10-py2.py3-none-any.whl"
-            },
-            {
-                "module": "requests",
-                "input_file": "wheels/shared/requests-2.25.0-py2.py3-none-any.whl"
-            },
-            {
-                "module": "soupsieve",
-                "input_file": "wheels/py3/soupsieve-2.3.2.post1-py3-none-any.whl"
-            },
-            {
-                "module": "urllib3",
-                "input_file": "wheels/shared/urllib3-1.26.11-py2.py3-none-any.whl"
-            },
-            {
-                "module": "validators",
-                "input_file": "wheels/py3/validators-0.20.0-py3-none-any.whl"
-            },
-            {
-                "module": "xmltodict",
-                "input_file": "wheels/shared/xmltodict-0.12.0-py2.py3-none-any.whl"
-            }
-        ]
-    }
-}
\ No newline at end of file
+{

```

Comment:
Would be nice to only show actual changes to this file

Reply:
you can toggle this client side: https://github.blog/2018-05-01-ignore-white-space-in-code-review/ (same thing on gitlab)

Reply:
Done